src/pspm_cfg/pspm_cfg_pp_heart_data.m

```diff
@@ -285,10 +204,10 @@
 ppg2hb_method.help  = {['Convert the PPG data into heart rate by using the selected method.']};
 
 ppg2hb              = cfg_exbranch;
-ppg2hb.name         = 'Convert peripheral pulse oximetry to Heart Beat';
+ppg2hb.name         = 'Convert peripheral pulse oximetry (PPG) to Heart Beat';
```

Comment:
I add `(PPG)` here because "peripheral pulse oximetry" is not a very common name.